google/cloud/security/common/data_access/sql_queries/select_data.py

```diff
@@ -149,11 +149,24 @@
     WHERE project_number = {1};
 """
 
-
-SELECT_VIOLATIONS = """
+SELECT_POLICY_VIOLATIONS = """
     SELECT * FROM violations_{0};
 """
 
+SELECT_BIGQUERY_ACL_VIOLATIONS = """
+    SELECT * FROM violations_{0}
+    WHERE violation_type = 'BIGQUERY_VIOLATION';
+"""
+
+SELECT_BUCKETS_ACL_VIOLATIONS = """
+    SELECT * FROM violations_{0}
+    WHERE violation_type = 'BUCKET_VIOLATION';
+"""
+SELECT_CLOUDSQL_VIOLATIONS = """
+    SELECT * FROM violations_{0}
+    WHERE violation_type = 'CLOUD_SQL_VIOLATION';
+"""
+
 SELECT_GROUPS_VIOLATIONS = """
     SELECT * FROM groups_violations_{0};
```

Comment:
Yes, group violations are in a separate table, because they have a different format. @blueandgold @gbrindisi